doc/apiary/v2/fiware-ngsiv2-rc-2018_04_30.apib

```diff
@@ -1780,7 +1780,7 @@ Notification rules are as follow:
 
 ## Subscription List [/v2/subscriptions]
 
-### Retrieve subscriptions [GET /v2/subscriptions{?limit,offset,options}]
+### List subscriptions [GET /v2/subscriptions{?limit,offset,options}]
```

Comment:
We have a mix of upper/lower case in the titles
We need to decide on the style to use

Reply:
For consistency the titles should be Title Case. It doesn't matter for **Swagger Code Gen** as this will be changed to `camelCase` automatically based on the whitespace between words.

* This is a title => `thisIsATitle`
* This title => `thisTitle`
* This Title => `thisTitle

Reply:
_Title Case_ sounds good to me

Reply:
NTC or Fixed in [commit]?

(Please have a look to the ["PR protocol"](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol) regarding how to end a comment thread before merging the PR. In this PR is not very important, as it is very simple, but better to get familiar with the procedure if you plan to contribute to the repo in a regularly basis ;). Thanks!)

Reply:
fixed in 517e23c